src/output/icons.rs

```diff
@@ -604,9 +620,10 @@ const EXTENSION_ICONS: Map<&'static str, char> = phf_map! {
     "rtf"            => Icons::TEXT,             // 
     "ru"             => Icons::LANG_RUBY,        // 
     "rubydoc"        => Icons::LANG_RUBYRAILS,   // 
-    "s"              => Icons::LANG_ASSEMBLY,    // 
+    "s"              => Icons::LANG_ASSEMBLY,    // 
```

Comment:
I'm not sure about this one tbh, \ue614 is for css

Reply:
Hi, is "s" an actual CSS file extension? Because I associate it with assembly language files. To be more specific I think they appear more often with a single capital "S" (like "`bootloader.S`"), so I could fix that. Like a C compiler will give you a ".S" assembly file if you do this: `$ cc -S program.c` -> `program.S`